### ISSUE TITLE
[Bug]: Model generate column field on oracle driver wrong

### ISSUE THREAD

**Issue by yonchando** [laravel-idea/plugin#1063]
### Bug description

In oracle all columns is UPPER when I generate model eloquent, it generate auto complete field with UPPER. I use model with UPPER field is return null and I use model with lower case it get data normal, but the IDE compliance it magic methods.

### Plugin version

8.2.5.242

### Operating system

Windows

### Steps to reproduce

Step 1:
Create model connection to oracle driver

Step 2:

Laravel Idea -> Generate Helper Code

Step 3:

Call model in controller with auto complete column field with UPPER or lower case

```php
$user = User::first();

echo $user->USERNAME; // auto complete from model generate helper but it return null

echo $user->username; // no auto complete and IDE compliance magic methods.
````

Expected

`$user->username`  Call username with lower from model with auto complete and IDE no compliance

Actual

`$user->USERNAME` USERNAME is auto complete from generate helper code and IDE know it field of model

**Comment by adelf** [comment]
Hello.

What do you use to fetch columns in Laravel Idea? Migrations or Database connection?

If migrations, could you show one of them? If connection, could you show the screenshot of the Database tab with some table columns?

**Comment by yonchando** [comment]
Hi,

I fetch columns from database connection.

![image](https://github.com/user-attachments/assets/d4513e50-ad3b-42c6-8e1e-e5f65ebbbfd5)

**Comment by adelf** [comment]
Ok. As you see, they are in upper case there. So, do you need only lowercase columns in the helper code?

**Comment by yonchando** [comment]
Yes, I tested it for Eloquent, If I call column field with uppercase, It get null and use lowercase it get data normal.

**Comment by yonchando** [comment]
It on where closure too. It use upper case. It could be lower case.

**Comment by yonchando** [comment]
Any updated?
